docs/docs/configuration/record.md

```diff
@@ -68,7 +68,20 @@ record:
 
 ## Will Frigate delete old recordings if my storage runs out?
 
-As of Frigate 0.12 if there is less than an hour left of storage, the oldest 2 hours of recordings will be deleted.
+As of Frigate 0.12 if there is less than an hour left of storage, the oldest 2 hours of recordings will be deleted. As of Frigate 0.15 this can be configured to trigger when free disk space drops below a specified minutes of recording time or raw disk space available. You can also specify the amount of disk space to free, either in recording time or raw disk space. When both `_minutes` and `_space` configuration values are specified, the larger calculated number is used.
+
+```yaml
+record:
+  cleanup:
```

Comment:
I don't think `cleanup` is a good name here. There is a separate cleanup process for recordings, the storage check there is a last ditch effort to ensure Frigate does not run out of space. Cleanup implies these are somehow related when they are not